src/pages/Trainee/TraineeList.jsx

```diff
@@ -22,23 +22,38 @@ class Trainee extends Component {
 
     this.state = {
       open: false,
+      selected: '',
+      orderBy: '',
+      order: '',
     };
   }
 
   openDialog = (status) => {
     this.setState({ open: status });
   };
 
-  // onClose = () => {
-  //   this.setState({ open: false }, () => { console.log(this.state); });
-  // };
-
   onSubmit = (data) => {
     this.setState({ open: false }, () => { console.log(data); });
   };
 
+  handleSort = (field) => (event) => {
+    const { order } = this.state;
+    this.setState({
+      orderBy: field,
+      order: order === 'asc' ? 'desc' : 'asc',
+    });
+  }
+
+  handleSelect = (event, data) => {
+    this.setState({ selected: event.target.value }, () => console.log(data));
+  };
+
+  Format = (date) => moment(date).format('dddd, MMMM Do YYYY, h:mm:ss a')
```

Comment:
Please use camelCase for method name as well
```suggestion
  format = (date) => moment(date).format('dddd, MMMM Do YYYY, h:mm:ss a')
```